docs/misc/faq.md

```diff
@@ -0,0 +1,110 @@
+---
+title: FAQ
```

Comment:
Some of these look like troubleshooting tips. Do we want to split them out to be separate? For FAQs/Troubleshooting, can/should we categorize related questions?